src/Interactive/VbiCore/project.lock.json

```diff
@@ -1175,33 +1175,6 @@
           "ref/dotnet5.4/Microsoft.Win32.Primitives.dll": {}
         }
       },
-      "runtime.any.System.Linq.Expressions/4.0.11-beta-23504": {
```

Comment:
The current version stated is this:

    src/Scripting/Core/project.json:14:    "System.Linq.Expressions": "4.0.10"`

So something else must have had a dependency pulling it forward that is no longer doing so.